src/app/modules/compose-email/compose-email.component.ts

```diff
@@ -37,7 +38,8 @@ export class ComposeEmailComponent implements OnInit {
     notGenuine: any;
     resendEmailTrackingData: boolean;
     holdSubject: any;
-    constructor(public setvardialog: MatDialog, public dialogRef: MatDialogRef<any>, private sendToManyEmail: ImapMailsService, public snackBar: MatSnackBar) {
+    parentId:any;
```

Comment:
what is the use of parentId and LocalStorageService initialization. Didn't found any related code.